client.go

```diff
@@ -692,8 +692,11 @@ type ToznySDKV3 struct {
 	// Tozny server defined globally unique id for this Client.
 	ClientID        string
 	CurrentIdentity TozIDSessionIdentityData
-	config          e3dbClients.ClientConfig
-	akCache         map[akCacheKey]e3dbClients.SymmetricKey
+	// AgentToken is the tozid-realm-idp jwt given by final redirect of login flow.
+	// because it expires, it is not saved to the config file, and so can be empty.
+	AgentToken *string
```

Comment:
I wouldn't worry about cross sdk consistency, if we can add more clarity with a more intention revealing name for the golang variables that would be good imo.

Reply:
"agent token" makes sense w/r/t tozid-realm-idp being an agent that logs you into tozid. but agreed. other options?
`SessionToken`, `TozIDRealmIDPAccessToken`?